src/axom/inlet/examples/nested_structs.cpp

```diff
@@ -333,9 +344,13 @@ int main(int argc, char** argv)
 
   if(docsEnabled)
   {
-    std::unique_ptr<inlet::SphinxWriter> writer(
+    std::unique_ptr<inlet::SphinxWriter> sphinxWriter(
       new inlet::SphinxWriter("nested_structs.rst"));
-    inlet.registerWriter(std::move(writer));
+    inlet.registerWriter(std::move(sphinxWriter));
+    inlet.writeDoc();
+    std::unique_ptr<inlet::JSONSchemaWriter> schemaWriter(
+      new inlet::JSONSchemaWriter("nested_structs.json"));
+    inlet.registerWriter(std::move(schemaWriter));
```

Comment:
(It will be easy for me to check, but) is there only one writer at a time?

Reply:
Yes (I had to look it up myself) - I don't see any reason why we couldn't just store a vector of them, though.

Reply:
https://github.com/LLNL/axom/issues/499

logged so we don't forget